docker/openqa_data/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM opensuse:13.2
+MAINTAINER Jan Sedlak <jsedlak@redhat.com> Josef Skladanka <jskladan@redhat.com>
+
+RUN zypper --non-interactive --gpg-auto-import-keys in ca-certificates git wget
```

Comment:
I have opened an issue for this in openSUSE/docker-containers repo here: https://github.com/openSUSE/docker-containers/issues/3.

Docker image doesn't include any certificates, so user can't curl from https and check if server can be trusted. Another problem is that image also doesn't have GPG keys of standard repositories imported, so user is also unable to install `ca-certificates` or `curl` packages without passing `--gpg-auto-import-keys`.

Currently, only solution is to run: `zypper --non-interactive --gpg-auto-import-keys in ca-certificates curl` at the beginning.